server/src/main/java/org/elasticsearch/index/mapper/RootObjectMapper.java

```diff
@@ -46,7 +46,7 @@
         public static final DateFormatter[] DYNAMIC_DATE_TIME_FORMATTERS =
                 new DateFormatter[]{
                         DateFieldMapper.DEFAULT_DATE_TIME_FORMATTER,
-                        Joda.getStrictStandardDateFormatter()
+                        DateFormatters.forPattern("yyyy/MM/dd HH:mm:ss||yyyy/MM/dd||epoch_millis")
```

Comment:
maybe we could name that pattern as well?

Reply:
sorry, not sure what do you mean with rename here? have an own method for it?

Reply:
previously we had `Joda.getStrictStandardDateFormatter()` I can see that DateFormatters is not declaring any public predefined constants for these patterns.
I was wondering if that would be beneficial to have some of these patterns predefined and named as contants
for instance we already have one here in `DateFieldMapper`:
`    public static final DateFormatter DEFAULT_DATE_TIME_FORMATTER = DateFormatters.forPattern("strict_date_optional_time||epoch_millis");
`
so following that we could add another one for STRICT_STANDARD_DATE
what do you think?